CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @awslabs/fhir-works
```

Comment:
Add a newline at the end of the file.

Not having it doesn't break anything(afaik), but it is a convention to have it and it makes git happy.